packages/libs/react-ui/src/styles/sprinkles.css.ts

```diff
@@ -73,10 +74,25 @@ const responsiveProperties = defineProperties({
     paddingRight: vars.sizes,
     marginTop: vars.sizes,
     marginBottom: vars.sizes,
-    marginLeft: vars.sizes,
-    marginRight: vars.sizes,
-    width: { ...vars.sizes, '100%': '100%', 'max-content': 'max-content' },
-    height: { ...vars.sizes, '100%': '100%' },
+    marginLeft: { ...vars.sizes, $auto: 'auto' },
```

Comment:
So this is a bit confusing, but we are not prefixing all values with `$`. It is only custom values. So in this case all of our custom sizes would be prefixed with `$` but since auto is one of the existing options for marginLeft, we would just name it `auto`. Also this should be added to the shorthand version `marginX`